pkgs/tools/security/ruler/default.nix

```
@@ -9,17 +10,24 @@ buildGoModule rec {
 
   src = fetchFromGitHub {
     owner = "sensepost";
-    repo = pname;
-    rev = version;
+    repo = "ruler";
+    rev = "refs/tags/${version}";
     hash = "sha256-cEYpK1LB9b65xr6MCMax1vUtSWefjJdXNs4sPgx65d0=";
   };
 
   vendorHash = "sha256-ITd3cvZmRBWK3922dDRvNHNH8KzHoVfIQI6S318ibxA=";
 
+  ldflags = [
+    "-w"
+    "-s"
+  ];
+
   meta = with lib; {
     description = "Tool to abuse Exchange services";
     homepage = "https://github.com/sensepost/ruler";
+    changelog = "https://github.com/sensepost/ruler/releases/tag/${version}";
     license = with licenses; [ cc-by-nc-40 ];
     maintainers = with maintainers; [ fab ];
+    mainProgram = "ruler";
```

I would suggest to remove the nested `with` scope.
Tracking issue: https://github.com/NixOS/nixpkgs/issues/292468

```suggestion
  meta = {
    description = "Tool to abuse Exchange services";
    homepage = "https://github.com/sensepost/ruler";
    changelog = "https://github.com/sensepost/ruler/releases/tag/${version}";
    license = with lib.licenses.cc-by-nc-40;
    maintainers = with lib.maintainers; [ fab ];
    mainProgram = "ruler";
```

---

Those change looks like something that should be done tree-wide and not on a per-package base.